src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -62,6 +62,7 @@
     <RootNamespace>SonarLint.VisualStudio.Integration.Vsix</RootNamespace>
     <AssemblyName>SonarLint.$(VersionSpecificSuffix)</AssemblyName>
     <RequiresSigning>true</RequiresSigning>
+    <RequiresShell>true</RequiresShell>
```

Comment:
When IssueViz.Security dependency is added to the Vsix project, it imports Shell 15 without an alias. Since Vsix already contains shell 14, the entire project is broken due to ambiguous assembly references.
So I "re-import" shell 15 with an alias as a way to silence the ambiguity.